app/scripts/modules/core/src/pipeline/status/ExecutionParameters.tsx

```diff
@@ -29,7 +25,7 @@ export class ExecutionParameters extends React.Component<IExecutionParametersPro
     if (parameter.valueTruncated) {
       return () => {
         parameter.showTruncatedValue = !parameter.showTruncatedValue;
-        this.setState({ toggle: parameter.showTruncatedValue });
+        this.forceUpdate();
```

Comment:
AFAICT this `setState` call was used exclusively to force a render when this parameter object was mutated, which gets confusing if you don't know that already. This is behaviorally the same but makes the intent more obvious.